execution_engine/src/core/runtime/mod.rs

```diff
@@ -2273,21 +2273,28 @@ where
         Ok(Ok(()))
     }
 
-    fn create_contract_package(&mut self) -> Result<(ContractPackage, URef), Error> {
+    fn create_contract_package(
+        &mut self,
+        is_locked: ContractPackageStatus,
```

Comment:
nitpick: you probably should have renamed the field to `status` when you changed it from a bool to a status enum.